x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps/handlers.ts

```diff
@@ -64,6 +101,11 @@ const errorHandler = <E extends Error>(
     return res.notFound({ body: error });
   }
 
+  if (error instanceof TrustedAppPolicyNotExistsError) {
+    logger.error(error);
+    return res.badRequest({ body: error });
```

Comment:
I was thinking we return a `404`, but after looking up the meaning of `400`, this is good 😄